storm-client/pom.xml

```diff
@@ -193,6 +193,10 @@
             <groupId>org.hamcrest</groupId>
             <artifactId>java-hamcrest</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-client</artifactId>
+        </dependency>
```

Comment:
Need to remove this.  It is a duplicate.  Not sure why my IDE wanted me to do this...